packages/widget-chat/src/components/activity-list/index.js

```diff
@@ -0,0 +1,58 @@
+import React, {Component, PropTypes} from 'react';
+import ActivityItem from '../activity-item';
+
+class ActivityList extends Component {
+  shouldComponentUpdate() {
+    return false;
+  }
+
+  renderActivityItems(activities) {
+    return activities.map((activity) =>
+      <ActivityItem activity={activity} key={activity.id} />
+    );
+  }
+
+  render() {
+    const activities = this.renderActivityItems(this.props.activities);
+    return (
+      <div>
+        <ul>
+          {activities}
+        </ul>
+      </div>
+    );
+  }
+}
+
+ActivityList.propTypes = {
+  activities: PropTypes.array,
+  user: PropTypes.object
+};
+
+ActivityList.defaultProps = {
+  activities: [
```

Comment:
I'm definitely up for not having the data model and the API bound. I just put very basic data on these components, knowing that we would be changing most of the params once we got data. Anything on this PR should be considered very basic placeholders.